usd/src/usd_parser/USDLight_TEST.cc

```diff
@@ -54,7 +54,7 @@ TEST(USDLightsTest, DistanceLight)
   ASSERT_TRUE(stage);
 
   pxr::UsdPrim prim = stage->GetPrimAtPath(
-    pxr::SdfPath("/shapes/defaultLight"));
+    pxr::SdfPath("/defaultLight"));
```

Comment:
Since we no longer have the `shapes` top level prim in the testing file, does the `schemaToStop` in the corresponding `ParseUSDLights` call need to be updated? It's currently `/shapes`, which doesn't exist: https://github.com/ignitionrobotics/sdformat/blob/a8249fbc63ce23b01a73df2344f0bcc63a4c9fab/usd/src/usd_parser/USDLight_TEST.cc#L60-L61